hummingbot/model/sql_connection_manager.py

```diff
@@ -5,6 +5,7 @@
 from os.path import join
 from sqlalchemy import (
     create_engine,
+    inspect
```

Comment:
incorrect syntax.

Reply:
thanks for noticing this part @martinkou